apps/web/vitest.config.ts

```diff
@@ -1,16 +1,17 @@
-import { defineVitestConfig } from 'nuxt-vitest/config';
+import { defineVitestConfig } from '@nuxt/test-utils/config';
 
 export default defineVitestConfig({
+  // any custom Vitest config you require
   test: {
     coverage: {
       reporter: ['text', 'html', 'json-summary', 'json'],
     },
     environment: 'nuxt',
-    clearMocks: true,
     globals: true,
-    setupFiles: './vue-test-utils.extend.js',
+    clearMocks: true,
+    setupFiles: './vitest.config.setup.ts',
     include: ['**/*/?(*.)+(spec|test).[jt]s'],
-    onConsoleLog: (log) => {
+    onConsoleLog: (log): any => {
       // Silence logs coming from vue <Suspense> is experimental, and stdout | unknown component before it
       if (log.includes('<Suspense')) return false;
       return log;
```

Comment:
```suggestion
    onConsoleLog: (log): false | void => {
      return silenceLogsFromSuspenseComponent(log);
```
**suggestion**: Specify return type and extract check to replace the inline comment.

```ts
const silenceLogsFromSuspenseComponent = (log: string): false | void => {
  if (log.includes('<Suspense')) return false;
};
```

Reply:
will change it